### ISSUE TITLE
Show a small hint window that email has been deleted/archived with an undo button.

### ISSUE THREAD

**Issue by Savasdotexe** [thundernest/k-9#6294]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.202

### Problem you are trying to solve

When you delete or achieve a message there isn't really anything that indicates you performed such an action (except when you delete, there is the confirm option), or if you have changed your mind, you would need to do a manual move to get it back.

A similar request was made at #624 for an undo option, but I'm not sure what happened.

### Suggested solution

Most modern email clients show a small fading hint that you've deleted or achieved that email. There could also be an undo button just in case you did it by accident or changed your mind. Changing the confirmation window to this fading hint would be more optimal and useful for both delete and archive. Those who prefer not to have to press again, but still want a notice.

### Screenshots / Drawings / Technical details

In the settings it could be something like "Enable hint window for deleted message" & "Enable hint window for achieved message".

**Comment by cketti** [comment]
Duplicate of #624

**Comment by Savasdotexe** [comment]
It's not really a duplicate, that thread suggests about an undo option instead of a confirmation for deletions, yes my post refers to an undo for delete, but I'd like one for **achieve** also and my suggestion is with a fading hint style popup..

In addition that thread was in 2015 with what looks like bickering occurring. Anyway, do what you like.

**Comment by woj-tek** [comment]
Small +1. I also found #624 but as mentioned by @Savasdotexe it referred to delete message and only later on it was dicussed in context of archive as well. What's more, it discusses pop-up with undo, which might work in a lot of cases, but it would be great to have "undo" entry in the menu.

Context: yesterday while trying to quickly open side drawer I accidentally swiped one of the messages (I already changed the swipe acction as it's way to error-prone to have "swipe right" to archive message as it could be errored with swiping the drawer way to often). It was kinda late so I didn't focus on the message so only noticed it disappeared.
